app/src/debug/AndroidManifest.xml

```diff
@@ -4,7 +4,9 @@
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
-    tools:ignore="ImpliedTouchscreenHardware,MissingLeanbackLauncher,MissingLeanbackSupport">
+    tools:ignore="ImpliedTouchscreenHardware,MissingLeanbackLauncher,MissingLeanbackSupport"
+    xmlns:amazon="http://schemas.amazon.com/apk/res/android"
+    package="org.mozilla.tv.firefox">
     <!-- ImpliedTouchscreenHardware, MissingLeanbackLauncher, and MissingLeanbackSupport
```

Comment:
I didn't notice this comment before and I think it applies here too: if it works, I think we should only ignore `GoogleAppIndexing` in the debug manifest. I think this is worth it because that means our change doesn't need to be temporary.

Sorry if my IRL comments mislead you :\